meetpy/templates/misc/home.html

```diff
@@ -24,9 +46,7 @@ <h4 class="date"><span class="day">{{ upcoming_meetup.date|date:"l" }}, </span>{
             <h5 class="location"><img src="{% static 'img/location.png' %}" alt="Lokalizacja" class="icon-location-home"> {{ upcoming_meetup.venue.name }}, {{ upcoming_meetup.venue.address }}</h5>
             <div class="text-center">
                 <div class="clearfix">
-                    <a href="https://maps.google.com/maps?q={{ upcoming_meetup.venue.latitude }},{{ upcoming_meetup.venue.longitude }}" target="_blank">
-                        <img src="https://maps.googleapis.com/maps/api/staticmap?zoom=15&amp;size=460x189&amp;maptype=roadmap&amp;markers=color:blue%7C{{ upcoming_meetup.venue.latitude }},{{ upcoming_meetup.venue.longitude }}&amp;sensor=false" class="map" alt="Mapa">
-                    </a>
+                    <div class="map" id="map-canvas"></div>
```

Comment:
może lat in lon możemy przekazać jako data-lat i data-lon do #map i wtedy cały ten skrypt razem z initialize() umieścić w tym includzie wyżej?